src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -1335,20 +1335,18 @@ public function prep_import_data( $data = array() ) {
 			return 'fetch';
 		}
 
-		$items = $this->filter_data_by_selected( $data->data->events );
-
-		return $items;
+		return $this->filter_data_by_selected( $data->data->events );
 	}
 
 	/**
 	 * Inserts events, venues, and organizers for the Import Record
 	 *
-	 * @param array $data Dummy data var to allow children to optionally react to passed in data
+	 * @param array $items Dummy data var to allow children to optionally react to passed in data
 	 *
 	 * @return Tribe__Events__Aggregator__Record__Activity The import activity record.
 	 */
-	public function insert_posts( $items = array() ) {
-		add_filter( 'tribe-post-origin', array( Tribe__Events__Aggregator__Records::instance(), 'filter_post_origin' ), 10 );
+	public function insert_posts( $items = [] ) {
```

Comment:
:warning: Function's nesting level (9) exceeds 5; consider refactoring the function

<sub>hash: 7d363d5f1c744a0ad7bae75c53e5cea4</sub>